### ISSUE TITLE
Create seminar content type

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#2588]
# Story

As a site owner I want to be able to list seminars/events on my website so that students or visitors can see what is coming up.

# Discussion

This is coming up now because the AMCS site has a list of seminars: https://www.amcs.uiowa.edu/amcs-seminars. I'm not sure if this is the right solution or whether it would be reasonable to suggest moving these to events.uiowa.edu. I'd appreciate any feedback. Also curious how you plan on handling this @mark-bennett-uiowa.

# Proposed solution

- [ ] Build an event/seminar content type
- [ ] Probably include the ability to link to an events.uiowa.edu event
- [ ] Create a list view of events/seminars
- [ ] Provide basic styling

**Comment by pyrello** [comment]
At this point, I have two sites that I was looking at this for. In both cases, we are going to encourage them to move their seminar content to events.uiowa.edu. So maybe this isn't necessary after all.

**Comment by pyrello** [comment]
@mark-bennett-uiowa Is this still something you are looking for? Or can I close this?

**Comment by mark-bennett-uiowa** [comment]
@pyrello Yes, unless we wanted to handle seminars/colloquia in Content Hub and the Events portal. Have you been successful in your work on this for the Grad College above?

The issue stems from private seminars (departmental audience only) vs public.

**Comment by pyrello** [comment]
We are pushing everyone into Content Hub. Here is an example on the AMCS site: https://amcs.uiowa.edu/amcs-seminars

I'm not sure I get your point about "private" seminars. If they are going to be posted on a public-facing website, then they are not private, even if the audience is more targeted. You can set the audience in Content Hub to be the program/department, I believe.

If you are going to have local seminar content, I think it would make sense to build this as part of the local event type rather than having a separate content type.
